src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -159,7 +160,7 @@ class ApplicationManagerImplTest
 #endif
     Mock::VerifyAndClearExpectations(mock_message_helper_);
   }
-  ~ApplicationManagerImplTest() {
+  ~ApplicationManagerImplTest() OVERRIDE {
```

Comment:
@IGapchuk why override?

Reply:
@AKalinich-Luxoft We need to mark this destructor as OVERRIDE, because of `ApplicationManagerImplTest` inherits `WithParamInterface` class, whose destructor is marked as virtual. And without OVERRIDE we have a compile-time warning.